tests/features/asset_release/asset_release.moderation.feature

```diff
@@ -0,0 +1,63 @@
+@api
+Feature: Asset release moderation
+  In order to manage releases
+  As a user of the website
+  I need to be able to transit the releases from one state to another.
+
+  Scenario: Publish, request changes and publish again a release.
+    Given the following owner:
+      | name        | type                  |
+      | Kenny Logan | Private Individual(s) |
+    And the following contact:
+      | name  | SheriMoore              |
+      | email | SheriMoore @example.com |
+    And users:
+      | name            | roles     |
+      | Bonnie Holloway |           |
+    And the following solutions:
+      | title     | description | logo     | banner     | owner       | contact information | state     |
+      | Dark Ship | Dark ship   | logo.png | banner.jpg | Kenny Logan | SheriMoore          | validated |
+    And the following solution user memberships:
```

Comment:
If I understood correctly, this is anyway connecting multiple tables so I think it's proper to use the column style only and you have updated the term to "Given the following solution user membership" by making the 's' optional. Right? 😆